events/lastAvatarStandingTrivia/clientScripts/trivia.html

```diff
@@ -95,43 +111,222 @@ <h3>Questions</h3>
+                            typeButton.disabled = true;
+                            diffButton.disabled = true;
+                            catButton.disabled = true;
+                            beginButton.disabled = false;
+                            endButton.disabled = true;
+                            newQButton.disabled = true;
+                            showQButton.disabled = true;
+                            answerButton.disabled = true;
+                        }       
+                    } catch (e){
+                        console.log(e, "error");
                     }
                 }
             }
 
             $(document).ready(function () {
                 EventBridge.scriptEventReceived.connect(onScriptEventReceived);
             });
+
+            var databaseButton = document.getElementById("catalog");
+            var typeButton = document.getElementById("selectedType");
+            var diffButton = document.getElementById("selectedDifficulty");
+            var catButton = document.getElementById("selectedCategory");
+            var endButton = document.getElementById("end");
+            var beginButton = document.getElementById("begin");
+            var newQButton = document.getElementById("newQuestion");
+            var showQButton = document.getElementById("showQuestion");
+            var answerButton = document.getElementById("showAnswers");
             $('#begin').click(function() {
                 var event = {
                     app: 'trivia',
                     type: "begin"
                 };
+                databaseButton.disabled = false;
+                typeButton.disabled = false;
+                diffButton.disabled = false;
+                catButton.disabled = false;
+                beginButton.disabled = true;
+                endButton.disabled = false;
+                newQButton.disabled = false;
+                document.getElementById('question').innerHTML = "The question will show here for preview.  <br><br> \
+                 * Try to avoid showing questions if either the question or answers use special characters.  <br><br> \
+                 ** Each answer should fit on a single line, as shown below.";
+                document.getElementById('answer').innerHTML = "Answer text should be no longer than this sentence."
                 EventBridge.emitWebEvent(JSON.stringify(event));
             });
             $('#end').click(function() {
                 var event = {
                     app: 'trivia',
                     type: "end"
                 };
+                databaseButton.disabled = true;
+                typeButton.disabled = true;
+                diffButton.disabled = true;
+                catButton.disabled = true;
+                endButton.disabled = true;
+                beginButton.disabled = false;
+                newQButton.disabled = true;
+                showQButton.disabled = true;
+                answerButton.disabled = true;
```

Comment:
the more of these kinds of grouped statements i see, the scarier the file gets! 😵